packages/svelte/src/internal/client/runtime.js

```diff
@@ -576,6 +584,35 @@ function process_microtask() {
 	}
 }
 
+const request_animation_frame =
+	typeof requestAnimationFrame === 'undefined' ? setTimeout : requestAnimationFrame;
```

Comment:
what's the reason for this? if it's to do with testing purposes then can we just do `globalThis.requestAnimationFrame = globalThis.setTimeout` in the tests?